charts/node/values.yaml

```diff
@@ -239,20 +239,35 @@ node:
   isParachain: false
 
   # -- EXPERIMENTAL!!!
-  # Run the collator node without a relay chain
+  # Run the collator node without a relay chain via external relay chain
   # ref: https://github.com/paritytech/cumulus#external-relay-chain-node
   # Enabling this option will disable the values of collatorRelayChain
-  externalRelayChain:
+  collatorExternalRelayChain:
     # -- Enable deployment of the external collator
     enabled: false
     # -- List of Relay Chain RPCs to connect
     relayChainRpcUrls: []
 
+  # -- EXPERIMENTAL!!!
+  # Run the collator node without a relay chain via light client
+  # ref: https://github.com/paritytech/cumulus/pull/2270
+  # Enabling this option will disable the values of collatorRelayChain
+  collatorLightClient:
+    # -- Enable deployment of the external collator
+    enabled: false
+    # -- Name of the Relay Chain to connect
+    relayChain: ""
+    # -- Path to the file containing the chainspec of the collator relay-chain
+    relayChainCustomChainspecPath: "/chain-data/relay_chain_chainspec.json"
+    # -- (string) URL to retrive custom chain spec
+    relayChainCustomChainspecUrl:
```

Comment:
`relayChain` and `relayChainCustomChainspecUrl` is already present:  `collatorRelayChain.chain`  `collatorRelayChain.customChainspecUrl`, it is very confusing without looking at the code why it is done like that. 
Since this is an experimental feature I don't mind having them duplicated. But for production usage, we should think of moving them under the `collatorRelayChain` section.:
```
.Values.node.collatorExternalRelayChain -> .Values.node.collatorRelayChain.externalRelayChain
.Values.node.collatorLightClient ->  .Values.node.collatorRelayChain.lightClient
```

Reply:
I disagree with moving them under `collatorRelayChain`. `collatoRelayChain` section is another mode to run a collator which runs both parachain and relaychain node in it while `collatorExternalRelayChain` and `collatorLightClient` are also another type to run a collator which is entirely independent to `collatorRelayChain`. 

My suggestion is to improve the naming and the fields we use. Something like:

```
collator:
  enabled: true
  
  relayChainNode:
    enabled: false
    All specific fields for relaychain node

  lightClientNode:
    enabled: false

  externalRelayChain:
     enabled: false
     rpcUrls: []
     
     
  chainSpec: xxx
  other shared fields: xxx
```

Reply:
> My suggestion is to improve the naming and the fields we use. Something like:
> ```
> collator:
>   enabled: true
>   
>   relayChainNode:
>     enabled: false
> ``` 

Now I see your point. Yes, your proposed mapping is better.  But since it will require a bumping a major version we can also rename the remaining fields. Something like:  

```
parachain:
  enabled: true
  
  embeddedRelayChainNode:
    enabled: false
    All specific fields for relaychain node

  lightClientRelayChainNode:
    enabled: false

  rpcRelayChainNode:
     enabled: false
     rpcUrls: []
     
     
  chainSpec: xxx
  other shared fields: xxx
```

Reply:
I agree that this would be a clearer config format (especially with @BulatSaif formulation). In the past we couldn't have anticipated that there would be so many "relay-chain node modes" in collators. Please propose this in a separate issue so we can discuss it.